packages/adblocker/src/filters/network.ts

```diff
@@ -1536,6 +1557,14 @@ export default class NetworkFilter implements IFilter {
       options.push('badfilter');
     }
 
+    if (this.isRemoveParam()) {
+      if (this.removeparam!.length > 0) {
+        options.push(`removeparam=${this.optionValue}`);
```

Comment:
Nit. Maybe we don't need to mix the use of three different methods/getters.
```suggestion
    const removeparam = this.removeparam;
    if (removeparam !== undefined) {
      if (removeparam.length > 0) {
        options.push(`removeparam=${removeparam}`);
```